woocommerce/woocommerce
# Add/35300 ces feedback product mvp

### All Submissions:






### Changes proposed in this Pull Request:

Hook in the CES modal to the share feedback button in the new product MVP
Closes #35300  .






### How to test the changes in this Pull Request:



1. Load this branch, build it, and make sure you have the `new-product-management-experience` feature flag enabled (you can do so using the latest version of the Beta tester within the mono repo).
2. Make sure you have tracks enabled and also outputted in the console: localStorage.setItem( 'debug', 'wc-admin:*' );
3. Go to Products > Add New (MVP)
4. In the top right click the 3 dots and click the **Share feedback** button
5. This should show the CES modal asking **How's your experience with the product editor?**
6. Answer it with your console open, this should trigger a `ces_feedback` track with the `new_product` action.
7. Now check if the CES footer still works by making sure the `woocommerce_ces_product_mvp_ces_action` option is removed
8. Save a product which should show the CES footer bar and click the **Share feedback** this should still correctly show the CES modal.



### Other information:




### FOR PR REVIEWER ONLY:
